README.md

```diff
@@ -155,7 +155,7 @@ able to build and install it just as you would the mainline Linux source.
 
 ### Building NOVA
 
-To build NOVA, build the kernel with PMEM (`CONFIG_BLK_DEV_PMEM`), DAX (`CONFIG_FS_DAX`) and NOVA (`CONFIG_NOVA_FS`) support.  Install as usual.
+To build NOVA, build the kernel with LIBNVDIMM (`CONFIG_LIBNVDIMM`), DAX (`CONFIG_FS_DAX`) and NOVA (`CONFIG_NOVA_FS`) support.  Install as usual.  (When running `make menuconfig`, you can find those options under the *Device Drivers* and *File Systems* sections, respectively.)
```

Comment:
Thanks for the pull request! But please keep CONFIG_BLK_DEV_PMEM as it is required by NOVA. Just add LIBNVDIMM.

Reply:
Thanks for the feeback!  I've updated my change.  In fact, enabling `LIBNVDIMM` in the menu automatically brings in `CONFIG_BLK_DEV_PMEM` as well.  But being verbose doesn't hurt anyway.